src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +16,29 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+
+        List<String> lines;
+        try {
+            lines = Files.readAllLines(Paths.get(fileName));
+        } catch (IOException e) {
+            e.printStackTrace();
+            return null;
```

Comment:
let's imagine i'm using this method and it's couldn't find a file and throw an exception, but program will continue to work and i will just get the null(not knowing that exception been thrown)

Reply:
done.
changed "catch" block